client/domain/FeedbackObjectFactorySpec.js

```diff
@@ -50,5 +50,7 @@ describe('FeedbackObjectFactory', function() {
       expect(feedback.getParagraphs.length).toEqual(0);
     });
   });
+
```

Comment:
I'm not sure why it's doing this but I reverted this file to a previous commit so it should be good now.